lib/JSON/Path.pm

```diff
@@ -49,7 +49,7 @@ class JSON::Path {
         self.bless(:$path);
     }
 
-    submethod BUILD(:$!path as Str) { }
+    submethod BUILD(Str :$!path) { }
```

Comment:
This was intended as a coercion rather than a constraint, so this should be `Str() :$!path`.